basic.js

```diff
@@ -15,9 +17,9 @@ const camera = new THREE.PerspectiveCamera(
     0.1,
     10900
 );
-camera.position.y = 1;
+camera.position.y = 0;
 
-const cameraLight = new THREE.SpotLight(0xffffff, 10);
+const cameraLight = new THREE.SpotLight(0xffffff, 1);
```

Comment:
Can you explain why did you modify camera position and light intensity?

Reply:
For the modification of y coordinate, it is just ease of view in my case.
And light was too much as the object surfaces can't be even distinguished.